Adding example. (#2)
Cogmasters/concord-docs: docs/source/discord/resources/channel/methods/get_channel.rst

===========

.. doxygenfunction:: discord_get_channel

Example
-------

.. code:: c
   
   struct discord_channel ret;

>    char text[128]

Reviewer: ```suggestion
```

   discord_get_channel(client, msg->channel_id, &ret);

>    snprint(text, sizeof(text), "The name of this channel is: %s", ret.name)

Reviewer: ```suggestion
   printf("The name of this channel is: %s", ret.name)
```

> 
>    struct discord_create_message_params params = { .content = text };
>       
>    discord_async_next(client, NULL);
>    discord_create_message(client, msg->channel_id, &params, NULL);
> 

Reviewer: ```suggestion
```
